src/Altinn.Platform/Altinn.Platform.Storage/DataCleanup/Services/BlobService.cs

```diff
@@ -29,10 +34,12 @@ public class BlobService : IBlobService
         /// </summary>
         /// <param name="logger">The logger.</param>
         /// <param name="sasTokenProvider">The sas token provider</param>
-        public BlobService(ILogger<IBlobService> logger, ISasTokenProvider sasTokenProvider)
+        /// <param name="keyVaultService">The key vault service</param>
+        public BlobService(ILogger<IBlobService> logger, ISasTokenProvider sasTokenProvider, IKeyVaultService keyVaultService)
```

Comment:
Maybe separate the operations related to the Backup storage in a new class?

Reply:
Makes it cleaner! Done.